packages/babel-plugin-transform-runtime/test/fixtures/runtime/modules-helpers/input.mjs

```diff
@@ -1,5 +1,6 @@
 import foo from "foo";
 
 class Example {
-  method() {}
+  property
```

Comment:
I rewrite this test case since it looks more practical that an ESM supported environment does not support class-properties and so we need to transform.